kafka-companion-frontend/src/app/topic/topic.component.html

```diff
@@ -1,8 +1,26 @@
 <div class="topic">
   <div class="topic-toolbar-area">
-    <kafka-toolbar [name]="topicName" (onAction)="onAction($event)"></kafka-toolbar>
+    <kafka-toolbar [name]="topicName" (onAction)="onAction($event)">
+    </kafka-toolbar>
+    <div class="kafka-topic">
+      <div class="kafka-topic-segment">
+        Choose partition:
+        <span *ngIf="partitions?.length > VISIBLE_PARTITION_QUANTITY" class="kafka-badge-secondary"
+              [ngClass]="{'kafka-badge-secondary-disabled': hasNoMorePrevValues()}" (click)="previous()">&#60;</span>
```

Comment:
na paginacji zastosowałem ikonki datatable-icon-left, datatable-icon-right. Myślę, że tutaj też możemy je zastosować

Reply:
w sumie racja, wydzielę to